packages/databricks-vscode/src/cli/CliWrapper.ts

```diff
@@ -284,6 +284,40 @@ export class CliWrapper {
         }
         return stdout;
     }
+
+    getBundleInitEnvVars(authProvider: AuthProvider) {
+        return {
+            ...EnvVarGenerators.getEnvVarsForCli(
+                workspaceConfigs.databrickscfgLocation
+            ),
+            ...EnvVarGenerators.getProxyEnvVars(),
+            ...this.getLogginEnvVars(),
+            ...authProvider.toEnv(),
+            DATABRICKS_OUTPUT_FORMAT: "text",
+        };
+    }
+
+    async bundleInit(
+        templateDirPath: string,
+        outputDirPath: string,
+        initConfigFilePath: string,
+        authProvider: AuthProvider
+    ) {
+        return await execFile(
+            this.cliPath,
+            [
+                "bundle",
+                "init",
+                templateDirPath,
+                "--output-dir",
+                outputDirPath,
+                "--config-file",
+                initConfigFilePath,
+            ],
+            {env: this.getBundleInitEnvVars(authProvider)}
+        );
+    }
```

Comment:
Maybe let's launch it in the same terminal as the normal bundle init? Just easier to debug when it fails, even for the user. And they have some visual feedback for automatic bundle init. Rn it seems like it is completely behind the scenes.

Reply:
We will show an error to the users eventually (when they go to the manual migration flow), but I think it makes sense to add error notification in the automatic migration flow (right now there's only error log).

Opening terminal makes sense for transparency, but it goes against the desire of not bothering existing project.json users.  Plus we will want to monitor command completion, which means custom terminal with pty, which adds a bunch of complexity